public/cli/lib/rendering/render.html

```diff
@@ -113,7 +113,7 @@
                         // concat the containers together for later
                         text += "<div class='words w" + j + "'>" + word + "</div> ";
                         // this makes the animation creation easier later
-                        wordTiming.push({word: word, index: j, start: wordStart, duration: wordDuration, line: i})
```

Comment:
Remove the creation of `wordDuration` on line 110 above since it is no longer used. Notice that lgtm.com bot is pointing that out here.

Reply:
@jamesduncan see above - thanks for this!